application/config/rest.php

```diff
@@ -187,6 +199,12 @@
 // ---Uncomment list line for the wildard unit test
 // $config['auth_override_class_method']['wildcard_test_cases']['*'] = 'basic';
 
+// JWT Exmaple
```

Comment:
So using these config is not a good idea, as they are used for something different.  We should not use them (maybe new config vars)